### PR TITLE
Additional Simplified Chinese translation

### PR DESCRIPTION
**Pull request by UM-Li**
Yes

### CODE REVIEW

File: src/musikcube/data/locales/zh_CN.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "schemaVersion": 1,
     "strings": {
-        "default_overlay_title": "musikcube",
+        "default_overlay_title": "Musikcube",
```
Comment (clangen) [inline src/musikcube/data/locales/zh_CN.json]: `musikcube` is actually branded completely lowercase. We should change the uppercase `M` back to `m` here and the few other places it was updated. :)

**Review by clangen — COMMENTED** [review]
Awesome, thanks so much for the update.

One small request: can we change `Musikcube` back to all lower-case `musikcube`? :)

**Comment by UM-Li** [comment]
Haha my mistake; thought about this but didn't confirm 😅

**Review by clangen — APPROVED** [review]
Thank you so much! This will be included in the release I'm currently working on!

**Pull request merged**